website/mantemplates/crowdin-distribution-add.adoc

```diff
@@ -35,4 +35,4 @@ Add a new distribution with the `bundle` export mode:
 crowdin distribution add "iOS Bundle" --export-mode bundle --bundle-id 19
 ----
 
-See the link:/crowdin-cli/commands/crowdin-bundle-list[`crowdin bundle list` command] to get the list of bundles including their IDs.
+See the link:/crowdin-cli/commands/crowdin-distribution-list[`crowdin distribution list` command] to get the list of distributions including their hashes.
```

Comment:
Actually, this was intentional :)

It's related to the example above with the `--bundle-id` parameter involved to tip the user where to find the ID of the bundle.

Reply:
Sorry, missed this